samples/openapi3/client/petstore/spring-cloud-oas3-fakeapi/src/main/java/org/openapitools/api/FakeClassnameTags123Api.java

```diff
@@ -31,7 +31,7 @@
 
 @Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
 @Validated
-@Tag(name = "FakeClassnameTags123", description = "the FakeClassnameTags123 API")
+@Tag(name = "fake_classname_tags 123#$%^", description = "the fake_classname_tags 123#$%^ API")
```

Comment:
same here. probably there are more places like these two

Reply:
Same as the other comment - this file already has a tag with that name on line 47.

I think these, and other, tests were written to ensure the name of the file, which is based on the tag name, is properly escaped. AFAICT there is no restriction on the tag name itself. The OpenAPI spec just describes it as a string. https://swagger.io/specification/#tagObject

Reply:
A picture speaks a thousand words. 

This is what version 6.2.1 renders. Note that the operation appear twice. Once under the tag that is based on the name of the API class (using the annotation at the class level), and under the tag as originally defined in the specification (using the annotation at the operation level):

With the PR, the updated FakeClassNameTagsApi is rendered as follows: as the class level and operation level tag are the same, the operation only appears the once - under the tag as defined in the specification:
![image](https://user-images.githubusercontent.com/435813/201299218-d845da96-872d-48bc-92ac-89a52514a59a.png)

Reply:
hmm. yeah, specification doesn't restrict allowed values for tag. happy to approve